### ISSUE TITLE
Error in Log since update to 3.0.0 (authentication related)

### ISSUE THREAD

**Issue by iHaveAstream** [dougiteixeira/proxmoxve#170]
### Checklist

- [X] I have updated to the latest available Proxmox VE Custom Integration version, [see the latest version here](https://github.com/dougiteixeira/proxmoxve/releases/latest).

### Describe the issue you are experiencing

Since update to v3.0.0 I get an Error log entry in HA log after each restart of HA:

`Error fetching proxmox_coordinator_update_proxmoxve data: User not allowed to access the resource, check user permissions as per the documentation.`

However, I haven't changed anything on Proxmox side and also not changed anything with regards to the dedicated user for this custom integration.
Definitely, this error comes up since the update to the latest version.

Moreover, I can observe that the integrations UI dialog now looks a bit weird:
![grafik](https://github.com/dougiteixeira/proxmoxve/assets/62019393/71c22dfb-3479-4b98-80c9-c0d3248d580e)

### In which version of Home Assistant Core do you have the problem?

2023.11.2

### What version of Proxmox VE Custom Integration has the issue?

3.0.0

### What version of Proxmox VE do you have the problem?

7.4-17

**Comment by dougiteixeira** [comment]
Can you download the diagnostic file ([see here how to download it](https://www.home-assistant.io/docs/configuration/troubleshooting/#download-diagnostics)) and send it here so I can analyze it better?


> Since update to v3.0.0 I get an Error log entry in HA log after each restart of HA:
> 
> `Error fetching proxmox_coordinator_update_proxmoxve data: User not allowed to access the resource, check user permissions as per the documentation.`

In this version, a new feature was added that requires additional permissions. A repair should have been created in your Home Assistant informing the necessary permissions.
See the permissions for each integration feature [here](https://github.com/dougiteixeira/proxmoxve#proxmox-permissions).





> Moreover, I can observe that the integrations UI dialog now looks a bit weird:
> ![grafik](https://user-images.githubusercontent.com/62019393/282848238-71c22dfb-3479-4b98-80c9-c0d3248d580e.png)

What language are you using in the UI? I'm also waiting for the diagnostic file to find a solution for this.

**Comment by iHaveAstream** [comment]
Hi!
Many thanks @dougiteixeira for your fast reply. Stupid me, typical "Layer 8" problem... actually I've notived that "Repair" info in HA, but I was too fast in clicking the "Ignore" button ;-)
I've now changed the group permissions for my HA integration user in Proxmox and now there is no more error or Repair info. 
So this is solved now!

Regarding the UI description issue, I'm using my HA with English UI language.
Diagnostics file attached.

[config_entry-proxmoxve-abff8ac75e16b3a70348e461a85a7185.json.txt](https://github.com/dougiteixeira/proxmoxve/files/13355484/config_entry-proxmoxve-abff8ac75e16b3a70348e461a85a7185.json.txt)
